File: PY/Bryann Valderrama/Matemática/primeFactors.py
from sys import stdout

def primeFactors(n):
    p = 2
    while p*p <= n:
        while n % p == 0:
            stdout.write(f'{p}\n')
            n //= p
        p += 1
    if n > 1:
        stdout.write(f'{n}\n')
        
primeFactors(20)

File: Codigos estudiantes por lenguaje/Java/Sebastian_Acevedo/Matematica/primeFactors.java
public class Main{
	
	static void primeFactors(int N) {
		for (int i = 2; i*i <= N; i++) {
			while(N%i==0) {
				System.out.println(i);
				N/=i;
			}
		}
		if(N>1)System.out.println(N);
	}
	
    public static void main(String[] args) {
    	primeFactors(5446);
    }  
}
